packages/devtools_app/lib/src/debugger/codeview.dart

```diff
@@ -606,22 +682,28 @@ class LineItem extends StatelessWidget {
               )
             ],
           ),
-          SelectableText.rich(
-            lineContents,
-            scrollPhysics: const NeverScrollableScrollPhysics(),
-            maxLines: 1,
-          ),
+          MouseRegion(
+              onExit: (_) => _onHoverExit(),
+              onHover: (e) => _onHover(e, context),
+              child: SelectableText.rich(
+                widget.lineContents,
+                scrollPhysics: const NeverScrollableScrollPhysics(),
+                maxLines: 1,
+              )),
         ],
       );
     } else {
-      child = SelectableText.rich(
-        lineContents,
-        scrollPhysics: const NeverScrollableScrollPhysics(),
-        maxLines: 1,
-      );
+      child = MouseRegion(
```

Comment:
this is duplicated code. Can you pull out into a local var and use in both places?